JoshuaOndieki/weconnect
# WeConnect UI templates

**What does this PR do?**
Adds UI templates for the WeConnect project.

**Description of Task to be completed?**
The front end UIs should be able to show how all the necessary processes of the WeConnect web app would be performed.

**Tasks achieved in this PR**


- [x] Login page

- [x] User homepage where logged in users are redirected to. Users can manage their businesses and view reviews feed on this page. The add and edit business are modal forms.

- [x] A business profile page where users can view the business info and also write more reviews.

- [x] A business listing page for discovering new businesses and easier search of the listed businesses.

**How should this be manually tested?**
Check out [joshuaondieki.github.io](https://joshuaondieki.github.io) and navigate through the UIs for manual testing. Alternatively, clone this repo and test locally on your computer.

**Resources & Libs**
The material UI by Google was used to accomplish this task. Specifically, I used [Materialize CSS](https://www.materializecss.com)
[This Pivotal Tracker board](https://www.pivotaltracker.com/n/projects/2153376) was used to plan and manage the tasks.
